### PR TITLE
Reduce image size

### PR DESCRIPTION
**Pull request by gongzhang**
This change will reduce the image size from 600+MB to 300+MB.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,15 +2,11 @@ FROM centos:7
 
 MAINTAINER <pahudnet@gmail.com>
 
-RUN yum update -y && \
-yum groupinstall "Development Tools"  -y && \
-yum install openssl-devel svn net-tools -y
-
-WORKDIR ['/root']
```
Comment (gongzhang) [inline Dockerfile:9]: `WORKDIR` does not support brackets. This will make wrong dir.

File: Dockerfile [hunk 2]
```diff
@@ -2,15 +2,11 @@ FROM centos:7
...
-
-RUN svn co https://shop.ntop.org/svn/ntop/trunk/n2n && \
-cd n2n/n2n_v2 && make && make install
-
-RUN yum groupremove "Development Tools"  -y && \
```
Comment (gongzhang) [inline Dockerfile:14]: Removing useless files on separate `RUN` won't reduce image size.

**Comment by gongzhang** [comment]
@pahud Please review the change. Thx :)

**Comment by pahud** [comment]
Thanks for the feedback.

I've used multi-stage Dockerfile to strip it down to 199MB

https://github.com/pahud/n2n-docker/commit/83ad399e2fcc90f2b76b78e6096bcd28c0ce01d4

**Comment by gongzhang** [comment]
That's great!

**Pull request closed without merging**